core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/UpdateHostsValidator.java

```diff
@@ -0,0 +1,128 @@
+package com.sequenceiq.cloudbreak.service.cluster.flow;
```

Comment:
I'm not sure what would be a good name, but I don't like something being called *FlowService and the methods are not really related to flow operations.